spec/reek/configuration/configuration_file_finder_spec.rb

```diff
@@ -42,6 +43,11 @@
       end
     end
 
+    it 'prefers the file in :current over one in :home' do
+      found = described_class.find(current: SAMPLES_PATH, home: SAMPLES_PATH.join('masked_by_dotfile'))
+      expect(found).to eq(SAMPLES_PATH.join('exceptions.reek'))
```

Comment:
No, `SAMPLES_PATH.join('exceptions.reek')` is in current. From home we would get `SAMPLES_PATH.join('masked_by_dotfile/.reek')`